src/main/kotlin/graph/Graph.kt

```diff
@@ -60,5 +60,10 @@ abstract class Graph<V> {
         weight: Int,
     )
 
+    abstract fun getShortestPath(
```

Comment:
We are going to implement two shortest path finding algorithms. Will that method manage ones usage?

Reply:
For the first time I rename this method
In the future, judging by the realization of Dijkstra's algorithm, we can rename it back or leave it as it is now